app/javascript/modules/listings/DirectoryHelpers.tsx

```diff
@@ -379,19 +379,19 @@ export const eligibilityHeader = (
       size: filters.household_size,
       people: filters.household_size === "1" ? t("listings.person") : t("listings.people"),
     })
-    const childrenContent = ` ${t("listings.includingChildren", {
+    const childrenContent = t("listings.includingChildren", {
       number: filters.children_under_6,
       children: filters.children_under_6 === "1" ? t("t.child") : t("t.children"),
-    })}`
-    const incomeContent = ` ${t("listings.atTotalIncome", {
+    })
+    const incomeContent = t("listings.atTotalIncome", {
       income: filters.income_total.toLocaleString(),
       per: getYearString(),
-    })}`
+    })
     return (
       <>
-        {renderInlineMarkup(householdSizeContent)}
-        {filters.include_children_under_6 && renderInlineMarkup(childrenContent)}
-        {renderInlineMarkup(incomeContent)}
+        {renderInlineMarkup(householdSizeContent, "<span>")}{" "}
+        {filters.include_children_under_6 && renderInlineMarkup(childrenContent, "<span>")}{" "}
+        {renderInlineMarkup(incomeContent, "<span>")}
```

Comment:
eek, you know how i feel about concatenating strings like this. these phrases may not be in the same order for all languages. ideally we would flatten this out.

that being said, i think you fixed the issue yindi found and afaik this is working for the 4 languages we currently support. will approve. i know this logic was carried over from the angular components